src/boot/efi/boot.c

```diff
@@ -477,7 +477,7 @@ static void print_status(Config *config, CHAR16 *loaded_image_path) {
             Print(L"           secure boot: %s (%s)\n", yes_no(IN_SET(secure, SECURE_BOOT_USER, SECURE_BOOT_DEPLOYED)), secure_boot_mode_to_string(secure));
           ps_bool(L"                  shim: %s\n",      shim_loaded());
           ps_bool(L"                   TPM: %s\n",      tpm_present());
-            Print(L"          console mode: %d/%d (%lux%lu @%ux%u)\n", ST->ConOut->Mode->Mode, ST->ConOut->Mode->MaxMode - 1LL, x_max, y_max, screen_width, screen_height);
+            Print(L"          console mode: %d/%ld (%" PRIuN L"x%" PRIuN L" @%ux%u)\n", ST->ConOut->Mode->Mode, ST->ConOut->Mode->MaxMode - INT64_C(1), x_max, y_max, screen_width, screen_height);
```

Comment:
I'd really prefer if we'd use PRIu32 style stuff for fixed size integers, like `screen_width` and `screen_height`. then it's much cloer to how we do stuff in userspace

Reply:
When I introduced this line you got paranoid about potential underflows should a crazy firmware give us a INT32_MIN as MaxMode. This is just continuing to cover our asses (and I didn't know at the time that `_C` macros existed or I would've used that). :P